pkg/util/mount/nsenter_mount.go

```diff
@@ -337,6 +337,9 @@ func (mounter *NsenterMounter) SafeMakeDir(subdir string, base string, perm os.F
 }
 
 func (mounter *NsenterMounter) GetMountRefs(pathname string) ([]string, error) {
+	if _, err := os.Stat(pathname); os.IsNotExist(err) {
```

Comment:
For nsenter, I think you need to call nsenter.ExistsPath.  This is a containerized kubelet environment where we cannot check paths in the kubelet container, we need to check paths on the host.

Reply:
Yes, you should call `mounter.ExistsPath()` to check if a path exists or not. Otherwise LGTM.